### PR TITLE
Feature/SM-2 Implemented the View Students Story

### PR DESCRIPTION
**Pull request by ProdanR**
Dear Lidia 
Please accept this pull request :)) smile face

### CODE REVIEW

File: src/loose/oose/fis/lab/student/manager/Main.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+import javafx.scene.Scene;
+import javafx.stage.Stage;
+
+public class Main extends Application {
+
+    Override
+    public void start(Stage primaryStage) throws Exception {
+        Parent root = FXMLLoader.load(getClass().getResource("fxml/view-students.fxml"));
+        primaryStage.setTitle("Student Manager");
+        primaryStage.setScene(new Scene(root, 600, 400));
```
Comment (szmecskas) [inline src/loose/oose/fis/lab/student/manager/Main.java]:
```suggestion
        primaryStage.setScene(new Scene(root, 600, 400));
```

File: src/loose/oose/fis/lab/student/manager/controllers/ViewStudentsController.java [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        studentMeanGradeColumn.setCellValueFactory(new PropertyValueFactory<>("meanGrade"));
+
+        studentTable.setItems(students);
+    }
+
+    private ObservableList<Student> students = FXCollections.observableArrayList(
+            new Student("John", "Doe", 20, 8.73),
+            new Student("Jane", "Does", 23, 9.35),
+            new Student("Jack", "Black", 19, 6.58),
+            new Student("Julia", "Nice", 21, 9.85),
```
Comment (szmecskas) [inline src/loose/oose/fis/lab/student/manager/controllers/ViewStudentsController.java]:
```suggestion
            new Student("Julia", "Nice", 21, 10),
```

File: src/loose/oose/fis/lab/student/manager/Main.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (ProdanR) [reply src/loose/oose/fis/lab/student/manager/Main.java]: Eu nu stiu sa ,,mofific,, sorry :((

Comment (ProdanR) [reply src/loose/oose/fis/lab/student/manager/Main.java]: Nu ai modificat nimic

**Pull request merged**